### PR TITLE
Docs: update backport commands for jenkins builds

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

**Comment by nebril** [comment]
test-backport-1.6

**Comment by nebril** [comment]
test-backport-1.7

**Comment by nebril** [comment]
test-backport-1.8

**Comment by nebril** [comment]
seems like new commands are working as intended

File: Documentation/contributing/release/backports.rst [hunk 1]
```diff
@@ -194,7 +194,7 @@ Running the CI against the pull request
 
 To validate a cross-section of various tests against the PRs, backport PRs
 should be validated in the CI by running all CI targets. This can be triggered
-by adding a comment to the PR with exactly the text ``never-tell-me-the-odds``.
+by adding a comment to the PR with exactly the text ``test-backport-x.x``, where ``x.x`` is target version.
```
Comment (pchaigno) [inline Documentation/contributing/release/backports.rst]: Nit: might not be worth re-pushing just for that.
```suggestion
by adding a comment to the PR with exactly the text ``test-backport-x.x``, where ``x.x`` is the target version.
```

**Review by pchaigno — APPROVED** [review]

**Review by tklauser — APPROVED** [review]
Nice! Should we change it in `Documentation/contributing/testing/ci.rst` (line 239ff):

```
For Backport PRs, the phrase ``never-tell-me-the-odds`` should be used to
trigger all of the above jobs which are marked as required to validate changes
to existing releases.
```

was well? Sorry, cannot link to the raw file line directly :confused:

**Comment by nebril** [comment]
@tklauser thanks, fixed that too!

**Review by tklauser — APPROVED** [review]
Added change in `ci.rst` also LGTM, thanks!

**Review by aanm — APPROVED** [review]

**Pull request merged**